app/code/Magento/CheckoutAgreementsGraphQl/etc/schema.graphqls

```diff
@@ -0,0 +1,15 @@
+# Copyright © Magento, Inc. All rights reserved.
+# See COPYING.txt for license details.
+
+type Query {
+    checkoutAgreements: [CheckoutAgreement] @resolver(class: "Magento\\CheckoutAgreementsGraphQl\\Model\\Resolver\\CheckoutAgreements") @doc(description: "The Checkout Agreements query returns information about a Checkout Agreements")
```

Comment:
`The Checkout Agreements query returns information about a Checkout Agreements`  :)